pom.xml

```
@@ -72,6 +72,7 @@
         <azure-core.version>1.21.0</azure-core.version><!-- @sync com.azure:azure-sdk-bom:${azure-sdk-bom.version} dep:com.azure:azure-core -->
         <azure-core-test.version>1.7.3</azure-core-test.version>
         <bouncycastle.version>1.70</bouncycastle.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:org.bouncycastle:bcprov-jdk15on -->
+        <c3p0.version>0.9.5.4</c3p0.version>
```

Maybe better to inherit the version from Camel.

```suggestion
        <c3p0.version>${3p0-version}</c3p0.version>
```

---

I used version from the camel, but it is a little bit higher, then it is used in quarkus. So I excluded mchange-commons from the io,quarkus (and kept slightly higfher version from camel)

The problems without exclusions were:
```
+-org.apache.camel.quarkus:camel-quarkus-quartz-deployment:2.8.0-SNAPSHOT
  +-io.quarkus:quarkus-quartz-deployment:2.7.3.Final
    +-io.quarkus:quarkus-quartz:2.7.3.Final
      +-org.quartz-scheduler:quartz:2.3.2
        +-com.mchange:mchange-commons-java:0.2.15
and
+-org.apache.camel.quarkus:camel-quarkus-quartz-deployment:2.8.0-SNAPSHOT
  +-org.apache.camel.quarkus:camel-quarkus-quartz:2.8.0-SNAPSHOT
    +-com.mchange:c3p0:0.9.5.5
      +-com.mchange:mchange-commons-java:0.2.19

```

```
Dependency convergence error for com.mchange:mchange-commons-java:0.2.15 paths to dependency are:
+-org.apache.camel.quarkus:camel-quarkus-quartz:2.8.0-SNAPSHOT
  +-io.quarkus:quarkus-quartz:2.7.3.Final
    +-org.quartz-scheduler:quartz:2.3.2
      +-com.mchange:mchange-commons-java:0.2.15
and
+-org.apache.camel.quarkus:camel-quarkus-quartz:2.8.0-SNAPSHOT
  +-com.mchange:c3p0:0.9.5.5
    +-com.mchange:mchange-commons-java:0.2.19

```